### ISSUE TITLE
Tooltip/Popover trigger property parameters are different from TWBS

### ISSUE THREAD

**Issue by Im0rtality** [angular-ui/bootstrap#2005]
$tooltip supports following triggers:
```
mouseenter: mouseleave
click: click
focus: blur
```
TWBS supports
```
click | hover | focus | manual
```

`mouseenter` vs `hover`

As per your contribution guide:
- `we are aiming at rebuilding bootstrap directives...`
- `directives should be html-agnostic...`

To avoid BC break, maybe it is possible to just add `hover` and mark `mouseenter` as deprecated?

**Comment by rvanbaalen** [comment]
@Im0rtality Any news on this?

**Comment by wesleycho** [comment]
I am not sure how wise it would be to mimic Bootstrap in this regard - this moves the consumer away from the standard JS events, for a bit of added complexity in the library code. The gain does not seem worth it IMO, but I don't feel too strongly either way.

**Comment by icfantv** [comment]
Closing as per #590, the user can now programmatically decide when to show and hide the tooltips/popovers.  This feature will be available in the 0.13.4 release and is available now on the master branch.
